js/main.js

```diff
@@ -9,50 +9,74 @@ var rejectBtn = document.querySelector('.reject-btn');
 
 var acceptBtn = document.querySelector('.accept-btn');
 
+var collectionView = document.querySelector('.collection-view');
+
 var image = '';
 
 function getLoadFox() {
-
   var xhr = new XMLHttpRequest();
   xhr.open('GET', 'https://randomfox.ca/floof/');
-
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
-
-    // console.log(xhr.status);
-    // console.log(xhr.response);
-
     summonImg.setAttribute('src', xhr.response.image);
     image = xhr.response.image;
-
     summonBtn.classList.add('hidden');
-
     acceptBtnDiv.classList.remove('hidden');
-
   });
-
   xhr.send();
-
 }
 
 function rejectFox() {
   summonBtn.classList.remove('hidden');
-
   acceptBtnDiv.classList.add('hidden');
-
   summonImg.setAttribute('src', 'images/pleasesummon.png');
 
 }
 
 function acceptFox() {
   summonBtn.classList.remove('hidden');
-
   acceptBtnDiv.classList.add('hidden');
-
   summonImg.setAttribute('src', 'images/pleasesummon.png');
-
   data.collection.push({ foxImage: image, quote: '' });
+}
+
+function collectionLoad(query) {
+
+  for (var i = 0; i < data.collection.length; i++) {
+    if (i < 3) {
+      var thirdColdiv = document.createElement('div');
+      thirdColdiv.classList.add('column-third');
+      thirdColdiv.classList.add('collection-item');
+      var collectionImg = document.createElement('img');
+      collectionImg.classList.add('collection-img');
+      collectionImg.setAttribute('src', data.collection[i].foxImage);
+      thirdColdiv.appendChild(collectionImg);
+      query.childNodes[1].appendChild(thirdColdiv);
+    } else if (i < 6) {
+      thirdColdiv = document.createElement('div');
+      thirdColdiv.classList.add('column-third');
+      thirdColdiv.classList.add('collection-item');
+      collectionImg = document.createElement('img');
+      collectionImg.classList.add('collection-img');
+      collectionImg.setAttribute('src', data.collection[i].foxImage);
+      thirdColdiv.appendChild(collectionImg);
+      query.childNodes[3].appendChild(thirdColdiv);
+
+    } else if (i < 7) {
+      thirdColdiv = document.createElement('div');
+      thirdColdiv.classList.add('column-third');
+      thirdColdiv.classList.add('collection-item');
+      collectionImg = document.createElement('img');
+      collectionImg.classList.add('collection-img');
+      collectionImg.setAttribute('src', data.collection[i].foxImage);
+      thirdColdiv.appendChild(collectionImg);
+      query.childNodes[5].appendChild(thirdColdiv);
+    } else {
+      break;
+    }
 
+  }
```

Comment:
Instead of making an individual row for every three foxes, can you make this one row with `row-wrap: wrap` and just add more children that can overflow to the next row? That will simplify this function by **_a lot_**

Reply:
Ok that worked alot faster than I thought it would.